dex-it/src/test/scala/com/wavesplatform/it/sync/api/MatcherWebSocketsTestSuite.scala

```diff
@@ -93,10 +97,40 @@ class MatcherWebSocketsTestSuite extends MatcherSuiteBase with HasWebSockets {
 
   "MatcherWebSocketRoute" - {
 
+    "should correctly handle connection rejections" in {
+
+      val kp = mkKeyPair("JIo6cTep_u3_6ocToHa")
+
+      val uriWithoutParams = s"${getBaseBalancesStreamUri(dex1)}${kp.publicKey}"
+      val uriWithParams    = s"$uriWithoutParams?t=${System.currentTimeMillis}&s=incorrect"
```

Comment:
It would be nice to have a test with the right signature too. E.g. interesting case when a signature is right, but an api key is not (it should definitely fail) or vise-versa (pass, because we ignore a signature)

Reply:
Done